### PR TITLE
PIM-5233 : merge 1.4 into master

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Extension/Pager/Orm/Pager.php [hunk 1]
```diff
@@ -101,7 +101,8 @@ public function computeNbResult()
         $rootField  = $rootAlias.'.id';
         $qb->groupBy($rootField);
 
-        $qb->setFirstResult(null)
+        $qb->select($rootField)
```
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Extension/Pager/Orm/Pager.php]: maybe we should wait to fix that on 1.4 before merge on master

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/change-family.html.twig [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {% extends 'PimEnrichBundle:MassEditAction:configure/layout.html.twig' %}
 
 {% block formContent %}
-    {{ form_row(form.operation.family) }}
+    <label class="control-label">{{ 'Family'|trans }}</label>
+    {{ form_row(
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/change-family.html.twig]: I don't like very much this syntax :(

File: src/Pim/Bundle/CatalogBundle/Resources/views/Collector/database.html.twig [hunk 3]
```diff
@@ -40,13 +40,13 @@
...
-            <p>Your current installation has {{ collector.getProductValueCount }} product values and support is enabled on your installation.</p>
+            <p>MongoDB is enabled on your installation.</p>
         {% else %}
             {% if collector.requireMongoDb %}
-                <p>Your current installation has {{ collector.getProductValueCount }} product values but MongoDB isn't installed yet. <br />
+                <p>Your current installation has more than 5M product values but MongoDB isn't installed yet. <br />
                 You should consider enabling MongoDB.</p>
             {% elseif not collector.requireMongoDb %}
-                <p>Your current installation has {{ collector.getProductValueCount }} product values and MongoDB isn't installed yet.<br />
+                <p>Your current installation has less than 5M product values and MongoDB isn't installed yet.<br />
```
Comment (pierallard) [inline src/Pim/Bundle/CatalogBundle/Resources/views/Collector/database.html.twig]: You can move the 2 exact same lines just after the {% else %}, no? Why are they not translated ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/meta/change-family-modal.html [hunk 4]
```diff
@@ -1,4 +1,12 @@
...
+    <%- _.__('pim_enrich.form.product.change_family.modal.keep_attributes')%>
+</p>
+<form class="form-horizontal">
+    <label class="control-label"><%- _.__('pim_enrich.form.product.change_family.modal.change_family_to')%>&nbsp;:</label>
+    <input type="hidden" class="input-large family-select2"
+           value="<%- product.family !== null ? product.family.code : '' %>"
+           <%- product.family ? ' selected' : '' %>
+           data-placeholder="<%- _.__('pim_enrich.form.product.change_family.modal.empty_selection') %>"
+    />
+</form>
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/meta/change-family-modal.html]: @jmleroux I though you have to refactor with this: https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/UIBundle/Resources/views/Form/pim-fields.html.twig#L533

**Comment by jmleroux** [comment]
closed because need rebase on 1.4 before

**Pull request closed without merging**